quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/CLITestExtension.java

```diff
@@ -76,17 +76,17 @@ public void afterEach(ExtensionContext context) throws Exception {
     public void afterAll(ExtensionContext context) throws Exception {
         if (dist != null) {
             // just to make sure the server is stopped after all tests
-            dist.stopIfRunning();
+            dist.stop();
+            dist.close();
         }
         super.afterAll(context);
     }
 
     private KeycloakDistribution createDistribution(DistributionTest config) {
-        KeycloakDistribution distribution = new KeycloakDistribution();
-
-        distribution.setReCreate(!ReInstall.NEVER.equals(config.reInstall()));
```

Comment:
Could you please elaborate why this config is no longer set to the raw distribution?